### DISCUSSION TITLE
Let's discuss a name for the new AsyncAPI design system!

### DISCUSSION THREAD

**Discussion by mcturco** [asyncapi/brand#26] [Ideas]
Good news! A design system for AsyncAPI is in the works 😎  For now, I am calling it `Design System` but I think we should ideate on a catchy name for it in this discussion. Some methods that we can use to come up with a name include the following: [(sourced from a Medium article on naming your design system)](https://medium.com/purplebricks-digital/how-to-name-your-design-system-722a83da287)

- Acronym
- Abbreviation
- Branded abbreviation
- Explicit naming
- Related proper noun
- Unrelated proper noun
- Unique descriptor
- [Adjective] + Design System
- [Noun] + Design System

Its very trendy right now to choose an unrelated proper noun (for example, [Shopify's Polaris](https://polaris.shopify.com/)) so I think that might be super fun to ideate on. 

Leave your ideas in the discussion below!

_Related to: #22_

**Comment by yashsehgal** [comment]
I have some suggestions for the names. 

- **Astral Design System** or **AsyncAPI's Astral**
- We can also try giving a real name to the design system such as: **Christine** (**AsyncAPI's Christine** or just **Christine**)

Hope you'll like these 😄

**Reply by mcturco** [reply]
**Astral** is a cool one! I enjoy it's "interstellar" theme. I think that picking a human name could be neat, but it might sound too personal. It would have to be an obscure name IMO!

**Reply by akshatnema** [reply]
Yupp Astral is giving a cool vibe as well. This name looks awesome.

**Comment by thehaupthaus** [comment]
- **Serial** by AsyncAPI
- **Signal** by AsyncAPI

**Reply by mcturco** [reply]
Nice ones! **Serial** definitely has a nod to software, and **Signal** is interesting because you need a signal to send and receive messages, which implies that our design system is necessary for our process. Thanks for the great feedback, Paul!

**Reply by yashsehgal** [reply]
Yes, **Serial** sounds good to me as well. Thanks for sharing @thehaupthaus

**Comment by magicmatatjahu** [comment]
Maybe `Elon` as our saviour Elon Musk? I know that it sounds like :trollface: but I'm serious here. Maybe someday even Elon will makea simple `AsyncAPI` (just that 8 letters) tweet and suddenly in 1 day we will have 10k stars on github 😆

**Reply by mcturco** [reply]
I agree, it could be a smart SEO move 😆
